app/common/helpers/collections.py

```diff
@@ -57,13 +57,15 @@ def name(self) -> str:
 
     @property
     def status(self) -> str:
+        submitted = MetadataEventKey.COLLECTION_SUBMITTED in [x.event_key for x in self.collection.collection_metadata]
```

Comment:
I maybe have a _tiny_ feel that some of this could live on the model itself as a `submitted` property, then we can just proxy `self.collection.submitted` or something - but this is fine for now. I think some of those core/simple/status-y values feel useful possibly outside of the helper. Also fine to wait until that is the case.

Reply:
This is a good idea, I'd like all the "reducers" for these to live in one place they could be unit tested but didn't think quite far ahead enough for how to structure that